### ISSUE TITLE
Error when archiving a series

### ISSUE THREAD

**Issue by cakarlen** [crunchy-labs/crunchy-cli#98]
**Describe the bug**
Hello, I'm attempting to archive the full series of `Fullmetal Alchemist: Brotherhood`. After downloading the first episode with the Japanese/English video track and subtitles, I receive the following error (bottom screenshot):

<img width="706" alt="CleanShot 2023-01-05 at 17 12 18@2x" src="https://user-images.githubusercontent.com/33635487/210890079-3035d183-d105-44de-9fa9-383686d2cc28.png">

<img width="1574" alt="CleanShot 2023-01-05 at 17 12 34@2x" src="https://user-images.githubusercontent.com/33635487/210890122-12d7c30c-af78-4091-916d-e79b08fc2509.png">

<img width="1582" alt="CleanShot 2023-01-05 at 17 09 20@2x" src="https://user-images.githubusercontent.com/33635487/210889679-6b47544b-6054-4277-91ec-f3a0b869a89c.png">

**To Reproduce**
Steps / command to reproduce the behavior:
```
$ crunchy-cli --credentials "username:password" archive -r best -o "S{season_number}E{episode_number} - {title}.mkv" --default-subtitle en-US "https://www.crunchyroll.com/series/GRGGPG93R/fullmetal-alchemist-brotherhood"
```

**Client (please complete the following information):**
 - OS: MacOS 12.3 (Monterey) M1
 - crunchy-cli 3.0.0-dev.6 (7726287 2023-01-05)
 - ffmpeg version N-109469-g62da0b4a74-tessus (built with Apple clang version 11.0.0 (clang-1100.0.33.17))

Is there any reason why this is happening? I can provide additional information as needed. Thanks!

**Comment by ByteDream** [comment]
Fixed in latest commit (892407d). This was caused because the temporary subtitle files were accidentally removed while setting which subtitle should be set as default.

**Comment by cakarlen** [comment]
Seems to have fixed the issue. Thanks for the fast fix
